src/style/common.scss

```
@@ -2,7 +2,7 @@ html,
 body,
 #root {
   width: 100%;
-  height: 100%;
+  min-height: 100%;
```

Привет!
Теперь из-за этого у нас липнут окошки к верху. См. тут и ниже, как пока лучше временно поменять.
```suggestion
  height: 100%;
```

```
body {
  background: url('images/main_bg.jpg') no-repeat;
  background-attachment: fixed;
  background-size: cover;
}
```

```
.main {
  ...
  height: 100%;
  ...
}
```